package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "lab-11-12-express-api",
+  "version": "1.0.0",
+  "description": "401 JS --  Lab 11 rest api using express ===",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "./node_modules/mocha/bin/mocha",
```

Comment:
Good job remembering test scripts. Remember to also include a nodemon script for future labs. :)